src/demo/coupons.js

```diff
@@ -49,34 +44,32 @@ const getAssignedTo = (isAssigned = false) => {
 
 const getCodes = (couponHasError = false) => {
   const redemptionsAvailablePerCode = faker.random.number({ min: 1, max: 5 });
-  return {
-    count: codesCount,
-    num_pages: Math.ceil(codesCount / 10),
-    current_page: 1,
-    results: [...Array(codesCount)].map((_, index) => {
-      const codeHasError = couponHasError && index <= 1;
-      const isAssigned = codeHasError || faker.random.boolean();
-      const assignedTo = getAssignedTo(isAssigned);
+  return [...Array(codesCount)].map((_, index) => {
+    const codeHasError = couponHasError && index <= 1;
+    const isAssigned = codeHasError || faker.random.boolean();
+    const assignedTo = getAssignedTo(isAssigned);
 
-      return {
-        title: Math.random().toString(36).substring(2).toUpperCase(),
-        assigned_to: assignedTo.name,
-        redemptions: {
-          available: redemptionsAvailablePerCode,
-          used: isAssigned ? faker.random.number({
-            min: 0,
-            max: codeHasError ? redemptionsAvailablePerCode - 1 : redemptionsAvailablePerCode,
-          }) : 0,
-        },
-        error: codeHasError ? `Unable to deliver email to ${assignedTo.name} (${assignedTo.email})` : null,
-      };
-    }),
-  };
+    return {
+      title: Math.random().toString(36).substring(2).toUpperCase(),
+      assigned_to: assignedTo.email,
+      redemptions: {
+        available: redemptionsAvailablePerCode,
+        used: isAssigned ? faker.random.number({
+          min: 0,
+          max: codeHasError ? redemptionsAvailablePerCode - 1 : redemptionsAvailablePerCode,
+        }) : 0,
+      },
+      error: codeHasError ? `Unable to deliver email to ${assignedTo.name} (${assignedTo.email})` : null,
+    };
+  });
 };
 
-coupons.results[0].has_error = firstCouponHasError;
+const getCodesCsv = () => 'assigned_to,code,redeem_url,redemptions.available,redemptions.used\r\n,ZBXF24QK6TJQMUSU,https://testserver.fake/coupons/offer/?code=ZBXF24QK6TJQMUSU,3,3\r\n,ZBXF24QK6TJQMUSU,https://testserver.fake/coupons/offer/?code=ZBXF24QK6TJQMUSU,3,3\r\n,ZBXF24QK6TJQMUSU,https://testserver.fake/coupons/offer/?code=ZBXF24QK6TJQMUSU,3,3\r\n,VF7TV7HMMQF63GS2,https://testserver.fake/coupons/offer/?code=VF7TV7HMMQF63GS2,3,3\r\n,VF7TV7HMMQF63GS2,https://testserver.fake/coupons/offer/?code=VF7TV7HMMQF63GS2,3,3\r\n,VF7TV7HMMQF63GS2,https://testserver.fake/coupons/offer/?code=VF7TV7HMMQF63GS2,3,3\r\n';
```

Comment:
Is there a way we can generate the CSV data string programmatically using the `getCodes` function instead of hardcoding it?